BP/common/landed_titles/BP_titles_turco_mongol.txt

```diff
@@ -13,18 +13,18 @@ e_hunnic_empire = {
 		}
 	}
 	
-	short_name = yes
+	# TODO: short_name = yes
```

Comment:
Same as for the Franks, `definite_form = yes`

Reply:
Fixed.